src/BugsnagServiceProvider.php

```diff
@@ -392,7 +394,9 @@ protected function setupSessionTracking(Client $client, $endpoint, $events)
 
         $genericStorage = function ($key, $value = null) use ($cache) {
             if (is_null($value)) {
-                return $cache->get($key, null);
+                $result = $cache->get($key, []);
```

Comment:
This is what's calling the test failure. The session tracker will utilise whatever is returned from this method in whatever format it's in. Previously this being null was fine by default, as we checked for nulls where necessary. We don't check in the SessionTracker (but maybe we should), as $a - null = $a.

Unfortunately the error being thrown isn't exactly visible due to being deep in docker.